Cargo.toml

```
@@ -166,6 +166,10 @@ tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 uuid = "1"
 x509-certificate = "0.23.1"
 
+[profile.optimized]
```

```suggestion
[profile.release-lto]
```

I think this is more idiomatic and explains what is actually happening. ref https://doc.rust-lang.org/cargo/reference/profiles.html#custom-profiles